locale/translations/pl_PL.json

```diff
@@ -1,13 +1,12 @@
 {
     "plural.feed.error_count": [
-        "%d błąd",

```

Comment:
There are 3 plural forms in Polish, you must have 3 items in this list even if the 2 first one are the same.